pkg/controller/installation/core_controller.go

```diff
@@ -727,6 +727,8 @@ func fillDefaults(instance *operator.Installation) error {
 			instance.Spec.Logging.CNI.LogFileMaxSize = new(resource.Quantity)
 			*instance.Spec.Logging.CNI.LogFileMaxSize = resource.MustParse("100Mi")
 		}
+	} else {
```

Comment:
I don't think we can do this - we can't just default away the configuration that a user might have set.

One of the core principles we follow is that we _never_ overwrite user input. Our code cannot be smart enough to know if this is the correct choice to make.